bin/_mocha

```diff
@@ -187,13 +187,44 @@ Error.stackTraceLimit = Infinity; // TODO: config
 
 var reporterOptions = {};
 if (program.reporterOptions !== undefined) {
-    program.reporterOptions.split(",").forEach(function(opt) {
-        var L = opt.split("=");
-        if (L.length != 2) {
-            throw new Error("invalid reporter option '" + opt + "'");
+
+   program.reporterOptions.split(",").forEach(
+
+    // Two possible formats ...
+     /\w+{(\w+:\w+)+}/.test(program.reporterOptions) ?
```

Comment:
@danielstjules I notice those files are on master, but not on v3.0.0.

Should I merge v3.0.0 up w/ master (or someone else)? Should I be targeting master instead?